src/main/java/frc/robot/subsystems/Lift.java

```diff
@@ -137,15 +143,39 @@ public CommandBase moveTicks(double ticks, double speed) {
                 CommandScheduler.getInstance().requiring(this).cancel();
             }
         }, () -> {
-            return Math.abs(liftEncoder.getDistance()) >= ticks;
+            return Math.abs(liftEncoder.getDistance()) >= Math.abs(0.10);
         }, this);
-        cmd.setName("Move Ticks");
+        cmd.setName("Move Disengage");
         return cmd.withTimeout(0.2);
     }
 
+    public CommandBase moveTicks(double ticks, double speed) {
+        CommandBase cmd = new FunctionalCommand(() -> {
+            liftEncoder.reset();
+        }, () -> {
+            double realSpeed = speed;
```

Comment:
This should be done once, outside of the functional command, instead of doing the math every cycle.